development/docker/install_latex.sh

```diff
@@ -0,0 +1,12 @@
+#Install latex direct from texlive
```

Comment:
It's usual to add a first line to `sh` scripts of the form `#! /bin/bash` to indicate what can be used to run it.